tf2_kdl/include/tf2_kdl/tf2_kdl.h

```diff
@@ -59,16 +62,46 @@ KDL::Frame transformToKDL(const geometry_msgs::TransformStamped& t)
  * \return The transform converted to a TransformStamped message.
  */
 inline
-geometry_msgs::TransformStamped kdlToTransform(const KDL::Frame& k)
+geometry_msgs::msg::TransformStamped kdlToTransform(const KDL::Frame& k)
 {
-  geometry_msgs::TransformStamped t;
+  geometry_msgs::msg::TransformStamped t;
   t.transform.translation.x = k.p.x();
   t.transform.translation.y = k.p.y();
   t.transform.translation.z = k.p.z();
   k.M.GetQuaternion(t.transform.rotation.x, t.transform.rotation.y, t.transform.rotation.z, t.transform.rotation.w);
   return t;
 }
 
+/** \brief Convert builtin_interfaces::msg::Time to the equivalent TimePoint type.
+ * \param t The builtin_interfaces::msg::Time to convert, as TimePoint type.
+ * \return The TimePoint conversion.
+ */
+inline
+tf2::TimePoint timePointFromTime(const builtin_interfaces::msg::Time& t)
```

Comment:
We already have versions of this and `timeFromTimePoint` in https://github.com/ros2/geometry2/blob/ros2/tf2_ros/include/tf2_ros/buffer_interface.h#L46 ; is there any reason we can't use those?

Reply:
Done, But I was wondering if this methods should go into `tf2/time.h`. inside `tf2_ros/buffer_interface.h` it's not very intuitive

Reply:
> Done, But I was wondering if this methods should go into `tf2/time.h`. inside `tf2_ros/buffer_interface.h` it's not very intuitive

The reason for that is that `tf2` itself is meant to be ROS-agnostic; you'll notice that there are no includes of ROS header files anywhere directly in tf2.  The interface between tf2 and ROS comes via the tf2_ros package.

That being said, I agree that it is not very intuitive that the time functions are in `tf2_ros/buffer_interface.h`.  I'd be in favor of making a new file `tf2_ros/time.h` where these methods would go, but I'll suggest we do that in a separate PR.

Reply:
Sure! I like `tf2_ros/time.h`